sstables/writer.hh

```diff
@@ -118,7 +123,7 @@ template <typename T>
 uint64_t
 serialized_size(sstable_version_types v, const T& object) {
     uint64_t size = 0;
-    auto writer = file_writer(make_sizing_output_stream(size));
+    auto writer = file_writer(make_sizing_output_stream(size), "<sizing_output_stream>");
```

Comment:
This is an indication that the file_writer isn't the correct place for the name, if you have to make up a fake name.

Reply:
The thing is that it's misnamed as it works on output streams that are not files.